test/pyc/.gitignore

```diff
@@ -0,0 +1 @@
+!*.pyc
```

Comment:
Forgot to add the pyc files, as they were gitignored (obviously), so added them now together with this.